ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/byok/cucushift-installer-rehearse-ibmcloud-ipi-byok-workflow.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/ibmcloud/ipi/byok/cucushift-installer-rehearse-ibmcloud-ipi-byok-workflow.yaml",
```

Comment:
```suggestion
	"path": "cucushift/installer/rehearse/ibmcloud/ipi/byo-kms/cucushift-installer-rehearse-ibmcloud-ipi-byo-kms-workflow.yaml",
```

Reply:
byo-kms ? or byo-kp?  kp-key?

Reply:
byo-kms, because:
1. the command is `ibmcloud resource service-instance-create ${instance_name} kms tiered-pricing ${region}`
2. also keep consistent with aws platform.

Reply:
cucushift/installer/rehearse/ibmcloud/ipi/byo-kms/cucushift-installer-rehearse-ibmcloud-ipi-**byo-kms-key-**workflow.yaml ?